docs/code/readme_template.md

```diff
@@ -61,7 +64,25 @@ path: /catalog/<COMPONENT_ID>/
 npm install @material/<MODULE_NAME>
 ```
 
-## Usage
+## Basic Usage
+
+<BASIC_USAGE_SECTION>
+
+## Variants
+
+<VARIANTS_SECTION>
+
+## Style Customization
+
+<STYLE_CUSTOMIZATION_SECTION>
+
+## `<JS_COMPONENT_NAME>` Properties and Methods
+
+<MDC_COMPONENT_SECTION>
+
+## Usage within Web Frameworks
+
+If you are using a JavaScript framework, such as React or Angular2, you can create a <COMPONENT_NAME> for your framework by wrapping <JS_COMPONENT_NAME>. Depending on your framework, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](https://github.com/material-components/material-components-web/blob/master/docs/integrating-into-frameworks.md).
```

Comment:
I might remove "by wrapping <JS_COMPONENT_NAME>" from the first sentence, since that sort of leads in one of the two possible directions whereas the second sentence explains both options. (Maybe also re-word "Depending on your framework" to "Depending on your needs" to reduce redundancy since the first sentence would also end with "framework" 3 words prior.)

Also, replace absolute link with a relative one i.e. `../integrating-into-frameworks.md`